tasks.php

```diff
@@ -322,12 +322,12 @@
 // -----------------------------------------------------------------------------
 
 $SearchParams_choices = array(
-    'task_status'   => array(998 => 'All Tasks', 999 => 'All Open Tasks') + $tasks_status_array,
-    'task_type'     => array(999 => 'All Task Types') + $tasks_array,
-    'task_severity' => array(999 => 'All Severities') + $severity_array,
-    'task_priority' => array(999 => 'All Priorities') + $priority_array,
-    'task_assignee' => array(999 => 'All Developers') + $task_assignees_array,
-    'task_category' => array(999 => 'All Categories') + $categories_array,
+    'task_status'   => array(998 => _('All Tasks'), 999 => _('All Open Tasks')) + $tasks_status_array,
```

Comment:
Should we translate some of these arrays also?

Reply:
I intentionally didn't do that (see my MR description). My reasoning at this time is that we will need to provide translators details on what each value means to have a meaningful translation and I'm not sure the value is worth the effort.